clustermesh-apiserver/users_mgmt.go

```diff
@@ -97,6 +97,7 @@ func (us *usersManager) Start(hive.HookContext) error {
 	}
 
 	us.manager.UpdateController(usersMgmtCtrl, controller.ControllerParams{
+		Group:   usersMgmtCtrl,
```

Comment:
I think for this component you would have to [explicitly enable the metric ](https://github.com/cilium/cilium/blob/main/clustermesh-apiserver/metrics/metrics.go#L75)

Reply:
Good to know. I'll have to confirm this.